helix-vcs/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "helix-vcs"
+version = "0.6.0"
+authors = ["Blaž Hrastnik <blaz@mxxn.io>"]
+edition = "2021"
+license = "MPL-2.0"
+categories = ["editor"]
+repository = "https://github.com/helix-editor/helix"
+homepage = "https://helix-editor.com"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+git-repository = {version = "0.23.1", default-features = false}
```

Comment:
Note that without the `max-performance-safe` feature, parallelism is also deactivated. At some point it might be noticeable that `ThreadSafeRepository` isn't actually `Send + Sync` anymore. If that ever becomes an issue, one would want `git-features/parallel` enabled, which can also be done by pulling it in separately. An example for this is [in `onefetch`](https://github.com/o2sh/onefetch/blob/94bfe2ed05175af0cdc84b0eef100e4527f215a2/Cargo.toml#L33).

Reply:
Thanks for pointing that out. I added the relevant feature flag because we already depend on the necessary creates anyway and while this PR does not need `ThreadSafeRepository`my followup PRs will (see below)